lib/lita/services/market_manager.rb

```diff
@@ -28,14 +28,14 @@ def placed_limit_order?(user_id)
         orders.map { |order| order['user_id'] }.include? user_id
       end
 
-      def add_market_order(lunch_sender_id)
-        return unless @karmanager.get_karma(lunch_sender_id) > 0
+      def add_market_order(lunch_buyer_id)
+        return unless @karmanager.get_karma(lunch_buyer_id) > 0
```

Comment:
Use @karmanager.get_karma(lunch_buyer_id).positive? instead of @karmanager.get_karma(lunch_buyer_id) > 0.